recipes/conversation_view/attachment_carousel/attachment_carousel.vue

```diff
@@ -0,0 +1,263 @@
+    as props to be passed down to another element or component using v-bind:$attrs
+    more info: https://vuejs.org/v2/api/#inheritAttrs */
+  // inheritAttrs: false,
+
+  props: {
+    /**
+     * media - object array of media objects
+     * @type {Array}
+     *
+     * Object: {
+     *  path: String,
+     *  altText: String | null,
+     * }
+     */
+    mediaList: {
+      type: Array,
+      default: () => [],
+    },
+
+    /**
+     * Descriptive label for the attachment element
+     */
+    attachmentAriaLabel: {
+      type: String,
+      required: true,
+      default: '',
+    },
+
+    closeAriaLabel: {
+      type: String,
+      required: true,
+      default: '',
+    },
+
+    clickToOpenAriaLabel: {
+      type: String,
+      required: true,
+      default: '',
+    },
+
+    leftArrowAriaLabel: {
+      type: String,
+      required: true,
+      default: '',
+    },
+
+    rightArrowAriaLabel: {
+      type: String,
+      required: true,
+      default: '',
+    },
+  },
+
+  emits: [
+    /**
+     * Emitted when popover is shown or hidden
+     *
+     * @event remove-image
+     * @type {Number}
+     */
+    'remove-image',
+  ],
+
+  data () {
+    return {
+      showCloseButton: {},
+      showCarouselArrows: false,
+      showRightArrow: true,
+      showLeftArrow: false,
+      isMounted: false,
+    };
+  },
+
+  computed: {
+
+  },
+
+  mounted: function () {
+    this.showLeftArrow = this.$refs.carousel.scrollLeft > 0;
+    this.showRightArrow = this.$refs.carousel.scrollWidth > this.$refs.carousel.clientWidth;
+  },
+
+  methods: {
+    removeMediaItem (index) {
+      this.$emit('remove-image', index);
+    },
+
+    closeButton (val, index) {
+      this.showCloseButton[index] = val;
+    },
+
+    handleScroll () {
+      const carousel = this.$refs.carousel;
+      this.showLeftArrow = carousel.scrollLeft > 0;
+      this.showRightArrow = !((carousel.scrollLeft + carousel.clientWidth) === carousel.scrollWidth);
+    },
+
+    leftScroll () {
+      this.$refs.carousel.scrollTo({
+        left: this.$refs.carousel.scrollLeft - 100,
+        behavior: 'smooth',
+      });
+    },
+
+    rightScroll () {
+      this.$refs.carousel.scrollTo({
+        left: this.$refs.carousel.scrollLeft + 100,
+        behavior: 'smooth',
+      });
+    },
+  },
+};
+</script>
+
+<style lang="less">
+.dt-attachment-carousel {
+  position: relative;
+  max-height: 100px;
+  width: var(--dt-space-1000);
+
+}
+
+.dt-attachment-carousel--media-list {
+  display: flex;
+  flex-direction: row;
+  overflow-x: scroll;
+}
+.dt-attachment-carousel--media-list::-webkit-scrollbar {
+  display: none;
+}
+
+.dt-attachment-carousel--close-button {
+  position: absolute;
+  color: var(--dt-color-neutral-white);
+  top: var(--dt-size-100);
+  right: var(--dt-size-100);
+  background-color: var(--dt-color-black-400);
+}
+.dt-attachment-carousel--left-arrow {
+  z-index: 100;
```

Comment:
Do you need z-index here at all or does it work without?

Reply:
works without it. removing.